proposals/20190909-psp-rules-support.md

```diff
@@ -0,0 +1,50 @@
+# Support for K8s Pod Security Policies (PSPs) in Falco
+
+<!-- toc -->
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  * [Use cases](#use-cases)
+  * [Diagrams](#diagrams)
+  * [Design Details](#design-details)
+
+<!-- tocstop -->
+
+## Summary
+
+We want to make it easier for K8s Cluster Operators to Author Pod Security Policies by providing a way to read a PSP, convert it to a set of falco rules, and then run falco with those rules.
```

Comment:
Yes! This would be amazing for the project! Strong +1 from me

Reply:
Where is the convertion done? In a cli ? In falco itself?

Reply:
It's in falco. Happy to discuss whether the conversion occurs in falco or outside--I lean towards in falco as it involves fewer moving parts, but obviously there are other opinions.

Remember, though, that the bulk of the work is in implementing the rules that result from the PSP, which requires new operators and fields to work on audit logs.